examples/dynamics/oscillator.ipynb

```diff
@@ -154,4 +156,4 @@
    "metadata": {}
   }
  ]
-}
```

Comment:
I can't even figure out how to display the raw source of a notebook within Github (you either get it rendered or the pure data without the Github interface).